crates/sui-indexer/src/schema.rs

```diff
@@ -286,14 +286,11 @@ macro_rules! for_all_tables {
             checkpoints,
             epochs,
             events,
-            events_partition_0,
```

Comment:
was this automatically or manually removed?

Reply:
Probably manually since partition 0's were pruned away?

Reply:
Manually (but this part is already updated manually).